daml-lf/repl/src/test/scala/com/digitalasset/daml/lf/ReplTest.scala

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+package speedy
+package testing
+import org.scalatest.wordspec.AnyWordSpec
+
+class ReplTest extends AnyWordSpec {
+
+  val file = "/tmp/Main.dar"
+
+  "Repl" should {
+
+    "succeed " in
+      assert(Repl.testAll(Repl.devCompilerConfig, file)._1)
+  }
+}
```

Comment:
I don’t understand what this is testing. Surely `tmp/Main.dar` is not present on CI?

Reply:
dropped.